lib/os-cluster-entrypoint.ts

```diff
@@ -134,6 +136,16 @@ export class OsClusterEntrypoint {
 
       const jvmSysProps = `${scope.node.tryGetContext('jvmSysProps')}`;
 
+      const osConfig = `${scope.node.tryGetContext('additionalConfig')}`;
+      if (osConfig.toString() !== 'undefined') {
+        try {
+          const jsonObj = JSON.parse(osConfig);
+          ymlConfig = dump(jsonObj);
+        } catch (e) {
+          throw new Error(`Encountered following error parsing json: ${e}`);
```

Comment:
nit: ```Error(`Encountered following error parsing json from additionalConfig: ${e}`);```

 Would make it easier for user to troubleshoot instead trying to figure out what json is wrong.

Reply:
didn't understand this. This is just catching exception in case the json is not valid. @gaiksaya